src/TypeSpec.Extension/Emitter.Csharp/test/TestProjects/lro/lro.tsp

```diff
@@ -47,5 +47,5 @@ model Pet {
 
 interface Projects {
     @doc("Creates a new pet or updates an existing one.")
-    createOrUpdate is LongRunningResourceCreateOrUpdate<Pet>;
+    createOrUpdate is LongRunningResourceCreateOrReplace<Pet>;
```

Comment:
Why do we update the test here? Have we decided not to support `LongRunningResourceCreateOrUpdate` ?

Reply:
See my comments at top in `Authoring-Typespec`. After discussing with Weidong, I decided to not support `LongRunningResourceCreateOrUpdate` until it's clarified. No real service is using it. So we're fine.